nekoyume/Assets/_Scripts/UI/Model/ItemInformationTooltip.cs

```diff
@@ -25,6 +25,7 @@ public class ItemInformationTooltip : Tooltip
 
         public readonly Subject<UI.ItemInformationTooltip> OnSubmitClick = new Subject<UI.ItemInformationTooltip>();
         public readonly Subject<UI.ItemInformationTooltip> OnCloseClick = new Subject<UI.ItemInformationTooltip>();
+        public readonly Subject<UI.ItemInformationTooltip> OnSubmitDisableClick = new Subject<UI.ItemInformationTooltip>();
```

Comment:
I personally think this variable's naming is not intuitive.
I would recommend `OnClickBlocked`.

Reply:
I thought the variable's name should follow `OnClickDisabledSubject` in class `ConditionalButton` .
But, In `ItemInformationTooltip`, Your advice would be better. Thank you!😃I fixed it.